FEXHeaderUtils/FEXHeaderUtils/Filesystem.h

```diff
@@ -200,6 +200,18 @@ namespace FHU::Filesystem {
     return false;
   }
 
+  /**
+   * @brief Renames a file and overwrites if it already exists.
+   *
+   * @param From
+   * @param To
+   *
```

Comment:
These aren't useful, so they can be removed.

Reply:
Removed.